### ISSUE TITLE
IP command found in ubuntu:16.04/latest

### ISSUE THREAD

**Issue by imesh** [wso2/dockerfiles#41]
It looks like the following error is raised in ubuntu:16.04

````
>docker run wso2esb:4.9.0
/usr/local/bin/init.sh: line 25: ip: command not found
mv: '/mnt/wso2esb-4.9.0' and '/mnt/wso2esb-4.9.0' are the same file
Creating directory /mnt/...
Moving carbon server from /mnt/wso2esb-4.9.0 to /mnt/...
````

**Comment by imesh** [comment]
Found the root cause, we need to install iproute2 (apt-get install iproute2) to use ip command now:
http://unix.stackexchange.com/questions/145447/ifconfig-command-not-found

**Comment by imesh** [comment]
Fixed with commit 3a402342d97d544c42f71ea7bb5ef4b2b1ed1a1c
